### DISCUSSION TITLE
Is STM32F412RGT6TR in supported SoC by zephyr os?

### DISCUSSION THREAD

**Discussion by SyedNasir112** [zephyrproject-rtos/zephyr#77336] [Q&A] [answered]
Is STM32F412RGT6TR included in the supported SoC? It comes under STM32F412xE/G devices.

**Comment by ndrs-pst** [comment, marked as answer]
According to this https://docs.zephyrproject.org/3.7.0/boards/mxchip/az3166_iotdevkit/doc/index.html, the short answer would be yes,
as this board includes the STM32F412RGT6TR pinctrl `st/f4/stm32f412r(e-g)tx-pinctrl.dtsi`.

In general, examining the SoC series device tree can give you a hint about which peripherals are supported or not. e.g. https://github.com/zephyrproject-rtos/zephyr/tree/8fcccf1aadae99f5341f20b4f305a37e953b17eb/dts/arm/st/f4

**Reply by SyedNasir112** [reply]
Thanks, It worked
